engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -23,15 +23,25 @@ import pl.touk.nussknacker.engine.api.CirceUtil._
   */
 @ConfiguredJsonCodec sealed trait ParameterValidator {
 
-  def isValid(paramName: String, value: String, label: Option[String])(implicit nodeId: NodeId): Validated[PartSubGraphCompilationError, Unit]
+  val priority: Long = 0
+  protected val skipIfBlank: Boolean
```

Comment:
In most cases it will be `true`. So mayby we can set it true? :) And mayby better name will be skipIfNotRequired?

Reply:
Ok, maybe you're right ;) But I think in most cases this skipping will be true.. @arkadius WDYT?

Reply:
and WDYT about 
```
//pseudo code
trait NotBlanParameterValidator extends ParameterValidator {
 def validateWhenNotBlank(value) 
 final def validate(value) = if(value.isBlank) Valid else validateWhenNotBlank(value)  
}
```
and then instead of setting `skipIfBlank = true` we could extend `NotBlankParameterValidator` and implement it?